### PR TITLE
fix: rename window title

### PR DESCRIPTION
**Pull request by keita-determined**
## Description

[DET-7776]

## Test Plan
1. Click `Uncategorized` tab in the side bar
1. Check the title on the tab

## Commentary (optional)

So far I just changed tab title because changing all file names and variables are too much.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-7776]: https://determinedai.atlassian.net/browse/DET-7776?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by trentwatt — COMMENTED** [review]
Looks like this changes all Projects, not just Uncategorized.

When we are actually in a Project, we would still want it to say "Project Details".

...or maybe the name of the project @alexjdetermined any thoughts?

image

**Comment by keita-determined** [comment]
> Looks like this changes all Projects, not just Uncategorized.
> 
> When we are actually in a Project, we would still want it to say "Project Details".
> 
> ...or maybe the name of the project @alexjdetermined any thoughts?
> 
> image

Hmm. It seems like both pages use the same component. I'll investigate it.

File: webui/react/src/pages/ProjectDetails.tsx [hunk 1]
```diff
@@ -931,7 +931,8 @@ const ProjectDetails: React.FC = () => {
     <Page
       bodyNoPadding
       containerRef={pageRef}
-      docTitle="Project Details"
+      // for docTitle, when id is 1 that means Uncategorized from webui/react/src/routes/routes.ts
```
Comment (keita-determined) [inline webui/react/src/pages/ProjectDetails.tsx:934]: The difference between `Uncategorized` and `Project Detail` are the project `id` in path, so I implemented this way. 
@trentwatt what do you think?
reference https://github.com/determined-ai/determined/blob/master/webui/react/src/routes/routes.ts#L32

**Review by trentwatt — APPROVED** [review]
LGTM.

**Pull request merged**